pubspec.yaml

```diff
@@ -7,5 +7,6 @@ authors:
 dev_dependencies:
   args: '0.13.7'
   dart_style: '>=1.0.8 <2.0.0'
+  io: '0.3.0'
```

Comment:
I don't think io package is not needed. What it does we can do with Process.run or Process.start, I think,

Reply:
The reason why I used `io` instead of `Process.run()` is that with the latter it's not possible to print the output while the command is running. For example, running the tests would display a blinking cursor for a few seconds, and then display the whole output at once. I then consulted with the Dart community, and one of the Dart devs from Google suggested using `io` which does the job perfectly.